docs/diagrams/CommandHistorySequenceDiagram.puml

```diff
@@ -0,0 +1,58 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":SociaiteParser" as SocialiteParser LOGIC_COLOR
+participant ":XYZCommand" as XYZcommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+
+box Storage STORAGE_COLOR_T1
+participant ":CommandHistoryStorage" as CommandHistoryStorage STORAGE_COLOR
+participant ":Storage" as Storage STORAGE_COLOR
+end box
+
+
+
+[-> LogicManager : execute(CMD)
+activate LogicManager
+
+LogicManager -> Model : addCommandToHistory(CMD)
+activate Model
+Model --> LogicManager
+deactivate Model
+
+LogicManager -> SocialiteParser : parseCommand(CMD)
+activate SocialiteParser
+SocialiteParser --> LogicManager : command
+deactivate SocialiteParser
+
+
+LogicManager -> XYZcommand : execute()
+activate XYZcommand
+XYZcommand --> LogicManager : result
```

Comment:
To be in line with the variable name used in `LogicManager.java`, do you want to change `result` to `commandResult` instead?

Reply:
Yep, I think it should be changed to `commandResult` also! Thanks for pointing that out!